src/general/SpeciesInfo.cs

```diff
@@ -0,0 +1,5 @@
+public class SpeciesInfo
```

Comment:
Could have a general explanation what SpeciesInfo is and what it could be used for. Beneficial for future programmers.